feature-libs/checkout/b2b/occ/config/default-occ-checkout-b2b-config.ts

```diff
@@ -1,12 +1,22 @@
+// We need these imports for augmentation of OccEndpoints to be picked up
```

Comment:
Reminder for https://github.com/SAP/spartacus/pull/14165#discussion_r749897883

Reply:
Switched all the libs to the new checkout entry point(s).
Please feel free to double check.

Reply:
I updated umd mappings for:
- cds
- dp
- organization administration components (I think)